.github/workflows/pr.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "Conventional PR and Commit Messages"
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+
+jobs:
+  semantic:
+    uses: influxdata/validate-semantic-github-messages/.github/workflows/semantic.yml@main
```

Comment:
What does this do exactly? What message format are we enforcing? Should we document this in the readme?

Reply:
This enforces https://www.conventionalcommits.org/en/v1.0.0/ yeah a note in the README is a good idea. We already use this style of commit message everywhere this just adds a CI check. Helps later if we want to automate generating changelogs.